### PR TITLE
documentation: ZENKO-195_OOB_updates_from_S3C

### PR DESCRIPTION
**Pull request by wabernat**
THIS PR updates Location Management section to help users understand
how to set up S3C. Doing this exposed the need for improvements across
all platforms, so I enhanced this setup step for all supported backends.

This PR does not entirely fix ZENKO-195, but is a step in the right direction!

### CODE REVIEW

**Comment by wabernat** [comment]
@bert-e reset
@bert-e approve

File: docs/docsource/Operation-Architecture/Orbit_UI/Location_Management/Adding_a_Storage_Location.rst [hunk 1]
```diff
@@ -37,12 +40,174 @@ Storage Locations dialog:
...
+
+Zenko provides a "Bucket Match" option for Ceph RADOS Gateway and
+Scality S3 Connector. If this option is left unchecked, Zenko prepends
+a bucket identifier to every object in the target backend's namespace.
+This enables a "bucket of buckets" architecture in which the target
+backend sees and manages only one large bucket and Zenko manages the
+namespace of the "sub-buckets." Clicking the **Bucket Match** box
+deactivates this feature: the prepending of bucket names is defeated,
+and the bucket structure in the host cloud is copied identically to
+the target cloud.
```
Comment (JianqinWang) [inline docs/docsource/Operation-Architecture/Orbit_UI/Location_Management/Adding_a_Storage_Location.rst:109]: Should we add a warning here that Bucket Match should not/cannot be used as a destination for CRR?

Comment (wabernat) [reply docs/docsource/Operation-Architecture/Orbit_UI/Location_Management/Adding_a_Storage_Location.rst:109]: Thank you! I will add that.

**Review by JianqinWang — APPROVED** [review]

**Review by ssalaues — APPROVED** [review]

**Comment by wabernat** [comment]
@bert-e approve

**Pull request merged**